src/planning/include/config/velocity_config.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef SRC_PLANNING_INCLUDE_CONFIG_VELOCITY_CONFIG_HPP_
+#define SRC_PLANNING_INCLUDE_CONFIG_VELOCITY_CONFIG_HPP_
+
```

Comment:
The "percentage" is defined in the launch file using the parameters normal_acceleration and braking_acceleration. If the car can withstand a normal acceleration of 15 m/s2 we can set how much we want to push it to its limit (for example: 66% of normal acceleration = 10m/s2). Using this aproach we can have a deeper understanding in where we are limiting the car (in cornering speed, or braking) instead of just a percentage. What do you think?